source/_integrations/waze_travel_time.markdown

```diff
@@ -69,36 +69,37 @@ template:
 
 In this example we use a device_tracker entity ID as the origin and the sensor created above as the destination.
 
-Name: "Me to destination"
-Origin: device_tracker.myphone
-Destination: sensor.dest_address
-Region: "US"
+  - Name: "Me to some destination"
+  - Origin: `device_tracker.myphone`
+  - Destination: `sensor.dest_address`
+  - Region: `US`
 
 #### Tracking entity to zone friendly name
 
 In this example we are using the entity ID of a zone as the origin and the friendly name of a zone as the destination.
 
-Name: "Home To Eddie's House"
-Origin: zone.home
-Destination: "Eddies House"
-Region: "US"
+  - Name: "Home to Eddie's house"
+  - Origin: `zone.home`
+  - Destination: `Eddies House`
+  - Region: `US`
 
 #### Tracking entity in Imperial Units
 
-Origin: person.paulus
-Destination: "725 5th Ave, New York, NY 10022, USA"
-Region: "US"
-Units: imperial
-Vehicle Type: motorcycle
+  - Name: "Somewhere in New York"
+  - Origin: `person.paulus`
+  - Destination: `725 5th Ave, New York, NY 10022, USA`
```

Comment:
We should not be using code fenced blocks for things that are not code.

Reply:
Do you have a suggestion on what to use instead? It's supposed to be an input string. I can use quotes instead if that makes more sense.